doc/mr.txt

```diff
@@ -36,6 +38,52 @@ Get |List| of MRU/MRW/MRR by calling |mr#mru#list()|, |mr#mrw#list()|, or
 	:echo mr#mrr#list()
 <
 
+How to use under fuzzy finder or auto completion plugin.
+
+CTRLP USAGE						*mr-usage-ctrlp*
+
+|ctrlp| is Pure Vim script fuzzy finder plugin.
+To use mr.vim with ctrlp, use ctrlp-mr.vim plugin and configure below:
+>
+	let g:ctrlp_extensions = get(g:, 'ctrlp_extensions', [])
+	        \ + ['mr/mru', 'mr/mrw', 'mr/mrr']
+<
+
+
+ASYNCOMPLETE USAGE					*mr-usage-asyncomplete*
+
+|asyncomplete| is Pure Vim script async auto completion plugin.
+To use mr.vim with asyncomplete, use asyncomplete-mr.vim plugin and configure
+below:
+>
+	" Setting to use MRU, same for MRW and MRR.
+	autocmd User asyncomplete_setup call
+	\ asyncomplete#register_source(
+	\   asyncomplete#sources#mru#get_source_options({
+	\     'name': 'mru',
+	\     'allowlist': ['*'],
+	\     'completor': function('asyncomplete#sources#mru#completor'),
+	\   }))
```

Comment:
Please follow Vim's default indentation rules. In other word, I prefer the code looks like the below

````vim
	autocmd User asyncomplete_setup call asyncomplete#register_source(
	      \ asyncomplete#sources#mru#get_source_options({
	      \   'name': 'mru',
	      \   'allowlist': ['*'],
	      \   'completor': function('asyncomplete#sources#mru#completor'),
	      \ })
	      \)
```

Reply:
Yes, reformat it.